### ISSUE TITLE
[YggTorrent] No result in TorrentPotato while some in manual search

### ISSUE THREAD

**Issue by na-ji** [Jackett/Jackett#3553]
**Jackett version**: 0.10.12.0
**Mono version** (if not using Windows): 5.12.0.301

-------------------------------
Hello!

In TorrentPotato, no result are returned while some are found in manual search with the same query. Here an example with a simple query:
In TorrentPotato
```
2018-08-08 14:03:36.8355 Info Request starting HTTP/1.1 GET http://127.0.0.1:9117/api/v2.0/indexers/yggtorrent/results/potato/?imdbid=tt2948356&search=Zootopia+2016&user=&passkey=key
2018-08-08 14:03:36.8355 Info Route matched with {action = "Potato", controller = "Results"}. Executing action Jackett.Server.Controllers.ResultsController.Potato (JackettConsole)
2018-08-08 14:03:36.8375 Info Executing action method Jackett.Server.Controllers.ResultsController.Potato (JackettConsole) with arguments (Jackett.Common.Models.DTO.TorrentPotatoRequest) - Validation state: Valid
2018-08-08 14:03:37.1064 Info Found 0 torrentpotato releases from YGGtorrent for: Zootopia 2016
2018-08-08 14:03:37.1064 Info Executed action method Jackett.Server.Controllers.ResultsController.Potato (JackettConsole), returned result Microsoft.AspNetCore.Mvc.ObjectResult in 269.4422ms.
2018-08-08 14:03:37.1064 Info Executing ObjectResult, writing value of type 'Jackett.Common.Models.DTO.TorrentPotatoResponse'.                                                                          
2018-08-08 14:03:37.1064 Info Executed action Jackett.Server.Controllers.ResultsController.Potato (JackettConsole) in 271.8453ms
2018-08-08 14:03:37.1064 Info Request finished in 272.8789ms 200 application/json; charset=utf-8
```

In manual search
```
2018-08-08 14:04:49.7288 Info Request starting HTTP/1.0 GET http://127.0.0.1:9117/api/v2.0/indexers/all/results?apikey=key&Query=Zootopia%202016&Tracker%5B%5D=yggtorrent&_=1533729859438
2018-08-08 14:04:49.7288 Info Route matched with {action = "Results", controller = "Results"}. Executing action Jackett.Server.Controllers.ResultsController.Results (JackettConsole)
2018-08-08 14:04:49.7288 Info Executing action method Jackett.Server.Controllers.ResultsController.Results (JackettConsole) with arguments (Jackett.Common.Models.DTO.ApiSearch) - Validation state: Valid
2018-08-08 14:04:50.0452 Info Manual search for "Zootopia 2016" on yggtorrent with 9 results.
2018-08-08 14:04:50.0452 Info Executed action method Jackett.Server.Controllers.ResultsController.Results (JackettConsole), returned result Microsoft.AspNetCore.Mvc.JsonResult in 314.2631ms.
2018-08-08 14:04:50.0452 Info Executing JsonResult, writing value of type 'Jackett.Common.Models.DTO.ManualSearchResult'.
2018-08-08 14:04:50.0452 Info Executed action Jackett.Server.Controllers.ResultsController.Results (JackettConsole) in 316.34ms
2018-08-08 14:04:50.0452 Info Request finished in 318.6721ms 200 application/json; charset=utf-8
```

Here are the enhanced logs for [torrent potato](https://pastebin.com/Gjri7p1s) and [manual search](https://pastebin.com/wkcvtVD7)

Thank you!

**Comment by na-ji** [comment]
Problem still occurs in 0.10.80.0

**Comment by kaso17** [comment]
Should be fixed via 6e59cc4c33ff3451ab33cd3cf03ac9b116e43daa
